docs/redux/redux-intro.md

```diff
@@ -150,132 +154,167 @@ Live demo: [http://jsbin.com/keyahus/edit?html,js,output](http://jsbin.com/keyah
 #### React
 
 ```js
-import {createStore} from 'redux';
-import PropTypes from 'prop-types';
-import {Component} from 'react';
-import ReactDOM from 'react-dom';
+import {createRoot} from 'react-dom/client';
+import {configureStore, createSlice} from '@reduxjs/toolkit';
```

Comment:
This example should not use redux toolkit since we didn't introduce it yet.
Please revert the change you did to use redux toolkit here.